using_openshift/output_deps.adoc

```diff
@@ -0,0 +1,53 @@
+= Dependency tree generation
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title: 
+
+toc::[]
+
+== Overview
+`build-chain` is an experimental command-line utility for OpenShift v3. It outputs the build dependency tree of a given image repository
+in JSON (default), DOT, or AST syntax. The user can specify an image repository name (and optionally a namespace and a tag) to build its
+dependency tree, call the tool without any arguments to build all dependency trees in the current namespace, or specify the `--all` flag to build all dependency trees in all available namespaces.
+ 
+
+== Examples
+
+
+Build dependency tree for the specified image repository with the specified tag in the current namespace and output it in JSON
+----
+$ openshift ex build-chain [image-repository]:[tag]
+----
+
+Build and output the dependency tree for the image repository in the specified namespace in DOT syntax
+----
+$ openshift ex build-chain [namespace]/[image-repository] -o dot
```

Comment:
also needs to indicate what happens if you do not specifiy a tag (assumes "latest" right?)